content/docs/intro/concepts/config.md

```diff
@@ -430,3 +430,61 @@ For example, this configures a stack to use a HashiCorp Vault transit key named
 $ pulumi stack init my-stack \
     --secrets-provider="hashivault://payroll"
 ```
+
+### Changing the Secrets Provider for a Stack
+
+> Note: There is not (yet) built-in `pulumi` CLI support for changing the secrets provider on an existing stack, but for cases where this is necessary, it is possible with some manual steps, as outlined in this section. 
+
+> Note: These steps can also be used to change the passphrase that is used for an existing stack.
+
```

Comment:
I think it would be helpful to list the high-level steps up front to make the detailed steps easier to follow along. Adding a section like this for example:

--------------------------

Here are the high-level steps needed to change the secrets provider for a stack with details below:
1. Export stack and config with secrets in plaintext
1. Initialize a temporary _new_ stack with the new encryption provider configuration
1. Modify your existing stack's state file with the configuration values of the new encryption provider
1. Re-import your state file with the updated encryption provider configuration